src/index.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  FunctionComponent,
+  Fragment,
+  ReactElement,
+  createElement,
+  useState,
+  useEffect,
+  useCallback,
+} from 'react';
+import unified, { PluggableList } from 'unified';
+import remarkParse, { RemarkParseOptions } from 'remark-parse';
+// @ts-ignore
+import remarkToRehype from 'remark-rehype';
+// @ts-ignore
+import rehypeReact from 'rehype-react';
```

Comment:
These packages don't have typings yet.
Work is being done upsteam to add typings.